dict-keys.py

```diff
@@ -9,8 +9,8 @@
     lin = lin.strip()
     words = lin.split()
     for wrd in words: #Loop that reads through each word
-        if wrd.startswith("From:"): #Reads through split words to identify if the line starts with 'From:'
-            wrd = lin.strip()
+        if wrd == "From": #Reads through split words to identify if the line starts with 'From:'
+            wrd = lin.strip() #strips whitespace

```

Comment:
Delete this line, I believe it's unnecessary to strip spaces within a list that has already been stripped from line 9.

Reply:
Makes sense, deleting line 13.